packages/docs/docs/plugin/official/plugin-pwa.md

```diff
@@ -46,7 +46,7 @@ module.exports = {
     ['meta', { name: 'msapplication-TileImage', content: '/icons/msapplication-icon-144x144.png' }],
     ['meta', { name: 'msapplication-TileColor', content: '#000000' }]
   ],
-  ['@vuepress/pwa', {
+  plugins: ['@vuepress/pwa', {
```

Comment:
should be:
`plugins: {
    '@vuepress/pwa': {
      serviceWorker: true,
      updatePopup: true
    }
  },`
or
`plugins: [
  ['@vuepress/pwa', {
      serviceWorker: true,
      updatePopup: true
  }]
],`